src/reactviews/pages/QueryResult/table/plugins/cellSelectionModel.plugin.ts

```diff
@@ -683,6 +684,12 @@ export class CellSelectionModel<T extends Slick.SlickData>
             isHandled = true;
         }
 
+        // Resize column (Shift+Alt+S)
+        if (keyCode === KeyCode.KeyS && e.shiftKey && e.altKey && !metaOrCtrlPressed) {
```

Comment:
Would be nice for readability if the checks were the same order as the shortcut description